README.md

```diff
@@ -79,9 +79,9 @@ For more information on the importance of a professional README for your reposit
 </details>
 
 <details>
-  <summary>CSSmary>
+  <summary>CSS</summary>
   <ul>
-    <li><a href="#">CSS</a></li>
+    <li><a href=""></a></li>
```

Comment:
- [ ] Your MIT license is broken. Please kindly fix and ensure it's working to have a professional README. Nice work.